velox/exec/PrefixSort.cpp

```diff
@@ -363,24 +471,37 @@ void PrefixSort::sortInternal(
   memory::ContiguousAllocation prefixBufferAlloc;
   // Allocates prefix sort buffer.
   {
+    // TODO: If we store the numNullRows of first column in RowContainer, we can
```

Comment:
I think this information is available now.
https://github.com/facebookincubator/velox/blob/main/velox/exec/RowContainer.h#L215